tests/compute/vmidefaults.go

```diff
@@ -243,8 +244,10 @@ var _ = SIGDescribe("VMIDefaults", func() {
 			Expect(err).ToNot(HaveOccurred())
 
 			By("Getting VirtualMachineInstance")
-			vmi, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(vm)).Get(context.Background(), vm.Name, metav1.GetOptions{})
-			Expect(err).ToNot(HaveOccurred())
+			Eventually(func() error {
+				vmi, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(vm)).Get(context.Background(), vm.Name, metav1.GetOptions{})
```

Comment:
Maybe it fits something like:
`Eventually(ThisVMI(vmi), 60).Should(BeInPhase(...))`

Reply:
Unfortunately, at that stage, we don't have any vmi object yet.

Reply:
You can use `ThisVMIWith`

Reply:
That fits. Thank you!

Reply:
Addressed. PTAL